packages/auth/src/app.ts

```diff
@@ -342,12 +344,29 @@ export class App {
 
     koa.use(cors())
     koa.keys = [this.config.cookieKey]
+
+    const redis = await this.container.use('redis')
+    const maxAgeMs = 60 * 1000
     koa.use(
       session(
         {
           key: 'sessionId',
-          maxAge: 60 * 1000,
-          signed: true
+          maxAge: maxAgeMs,
+          signed: true,
+          store: {
+            async get(key) {
+              const data = await redis.get(key)
+              return data ? JSON.parse(data) : null
+            },
+            async set(key, session) {
```

Comment:
I see there is a third parameter `maxAge` possible in this function. Do we want to include that here?